docs/components/generic/_index.md

```diff
@@ -83,6 +83,13 @@ import (
 )
 ```
 
+{{% /tab %}}
+{{% tab name="C++" %}}
+
+```cpp
+#include <viam/sdk/services/generic/generic.hpp>
```

Comment:
```suggestion
#include <viam/sdk/components/generic/generic.hpp>
```
I think you're wanted to show the `#include` for the generic component here, right? Not the service?

Reply:
Sorry: this change is an out-of-scope edit to the generic _component_ reference page, which didn't mention C++ at all. Now, with this edit, it does!